### PR TITLE
chore: add dependabot for npm and github-actions

### PR DESCRIPTION
**Pull request by lsaudon**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Add dependabot for automates dependency updates.

### CODE REVIEW

**Comment by SeparateRecords** [comment]
@fedeci I don't have any particular disagreement with adding dependabot. Doesn't strike me as _necessary_ per se, but it couldn't hurt to have our dependencies kept up to date automatically. Thoughts?

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```
Comment (fedeci) [inline .github/dependabot.yml:10]: Having it daily is quite annoying when you have a lot of deps.
```suggestion
      interval: 'weekly'
```

**Comment by fedeci** [comment]
@SeparateRecords I agree with you. We just need to check the changelogs when we bump the version of packages. It happens that even patches break stuff.

**Comment by SeparateRecords** [comment]
Sounds good. Will test changes locally before merging too, can check for build errors!

**Review by SeparateRecords — APPROVED** [review]
Thanks @lsaudon !

**Pull request merged**